config/routes.rb

```diff
@@ -2,4 +2,6 @@
   root to: 'home#index'
 
   resources :items, only:[:new, :create, :index]
+
+  get '/:category', to: 'categories#show', param: :slug
```

Comment:
Nice! I'm glad you went with this because that's what the spec calls for. You'll have to show me in the morning how it all came together.